### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by myoung34**
without this change, the installer fails with

```
myoung@bigNASty:~$ sudo cat /var/log/packages/nomad.log
2019/05/30 06:18:05     install nomad 0.9.1 Begin preinst
cp: cannot stat '/usr/local/nomad/nomad/conf.d': No such file or directory
2019/05/30 06:18:05     install nomad 0.9.1 End preinst ret=[0]
2019/05/30 06:18:05     install nomad 0.9.1 Begin /bin/mv -f /volume1/@tmp/pkginstall/package /volume1/@appstore/nomad
2019/05/30 06:18:05     install nomad 0.9.1 End /bin/mv -f /volume1/@tmp/pkginstall/package /volume1/@appstore/nomad ret=[0]
2019/05/30 06:18:05     install nomad 0.9.1 Begin /bin/rm -rf /var/packages/nomad
2019/05/30 06:18:05     install nomad 0.9.1 End /bin/rm -rf /var/packages/nomad ret=[0]
2019/05/30 06:18:06     install nomad 0.9.1 Begin /bin/mkdir -p /var/packages/nomad
2019/05/30 06:18:06     install nomad 0.9.1 End /bin/mkdir -p /var/packages/nomad ret=[0]
2019/05/30 06:18:06     install nomad 0.9.1 Begin /bin/mv -f /volume1/@tmp/pkginstall/INFO /var/packages/nomad/INFO
2019/05/30 06:18:06     install nomad 0.9.1 End /bin/mv -f /volume1/@tmp/pkginstall/INFO /var/packages/nomad/INFO ret=[0]
2019/05/30 06:18:06     install nomad 0.9.1 Begin /bin/rm -rf /var/packages/nomad/scripts
2019/05/30 06:18:06     install nomad 0.9.1 End /bin/rm -rf /var/packages/nomad/scripts ret=[0]
2019/05/30 06:18:06     install nomad 0.9.1 Begin /bin/mv -f /volume1/@tmp/pkginstall/scripts /var/packages/nomad/scripts
2019/05/30 06:18:06     install nomad 0.9.1 End /bin/mv -f /volume1/@tmp/pkginstall/scripts /var/packages/nomad/scripts ret=[0]
2019/05/30 06:18:06     install nomad 0.9.1 Begin /bin/rm -rf /var/packages/nomad/conf
2019/05/30 06:18:06     install nomad 0.9.1 End /bin/rm -rf /var/packages/nomad/conf ret=[0]
2019/05/30 06:18:06     install nomad 0.9.1 Begin /bin/mv -f /volume1/@tmp/pkginstall/conf /var/packages/nomad/conf
2019/05/30 06:18:06     install nomad 0.9.1 End /bin/mv -f /volume1/@tmp/pkginstall/conf /var/packages/nomad/conf ret=[0]
2019/05/30 06:18:06     install nomad 0.9.1 Begin postinst
mkdir: cannot create directory '/usr/local/nomad/nomad': Not a directory
2019/05/30 06:18:06     install nomad 0.9.1 End postinst ret=[0]
2019/05/30 06:18:06     install nomad 0.9.1 Begin /bin/rm -rf /volume1/@tmp/pkginstall
2019/05/30 06:18:06     install nomad 0.9.1 End /bin/rm -rf /volume1/@tmp/pkginstall ret=[0]
2019/05/30 06:18:06     install nomad 0.9.1 Begin start-stop-status start
Starting nomad ...
2019/05/30 06:18:06     install nomad 0.9.1 End start-stop-status start ret=[0]
```

Namely `cp: cannot stat '/usr/local/nomad/nomad/conf.d': No such file or directory`
thats because `/usr/local/nomad/nomad` is a file (the nomad binary) not a dir

```
myoung@bigNASty:~$ sudo cat /var/log/nomad.log
Using configuration file in
/var/packages/nomad/scripts/start-stop-status: line 32: /usr/local/nomad/nomad/nomad: Not a directory
Using configuration file in
/var/packages/nomad/scripts/start-stop-status: line 32: /usr/local/nomad/nomad/nomad: Not a directory
```

This change fixes that as well as hydrates the README a bit

After this change I verified everything is great:

```
$ nomad node status | grep bigNASty
4a8913cd  home  bigNASty  <none>  false  eligible     ready
```

### CODE REVIEW

**Comment by numkem** [comment]
Where do you put your binary? I don't have any of these issues myself.

File: 2_create_project/INFO [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-version="0.10.0"
+version="0.12.4"
 description="Easily Deploy Applications at Any Scale"
 support_url="https://github.com/hashicorp/nomad/"
 displayname="nomad"
 maintainer="hashicorp"
 maintainer_url="https://www.nomadproject.io"
-distributor="Sebastien Bariteau"
-distributor_url="https://github.com/numkem/nomad-spk"
+distributor="Marcus Young"
```
Comment (numkem) [inline 2_create_project/INFO]: Could these lines be removed?

File: 1_create_package/nomad/conf.d/server.conf [hunk 2]
```diff
@@ -3,8 +3,6 @@ bind_addr = "0.0.0.0" # the default
 
 advertise {
   # Defaults to the first private IP address.
-  http = "192.168.2.113"
```
Comment (numkem) [inline 1_create_package/nomad/conf.d/server.conf]: I don't have this file at all in my repo. Is that something new?

**Review by numkem — CHANGES_REQUESTED** [review]

File: 1_create_package/nomad/conf.d/server.conf (hunk 2 above)
Comment (myoung34) [reply 1_create_package/nomad/conf.d/server.conf]: It is, i've diverged pretty heavily, some of this was cruft from testing

File: 2_create_project/INFO (hunk 1 above)
Comment (myoung34) [reply 2_create_project/INFO]: done

File: 2_create_project/scripts/start-stop-status [hunk 3]
```diff
@@ -19,7 +19,8 @@ if [ -d "${ALT_CONF_DIR}" ];
 then
     CONF_DIR="${ALT_CONF_DIR}"
 fi
-ARGS="agent -config=${CONF_DIR}"
+CONF_DIR="/volume1/@appstore/nomad/nomad/conf.d/server.conf"
```
Comment (myoung34) [inline 2_create_project/scripts/start-stop-status:22]: this was blowing up on `DIR_NAME` not being set during testing

Comment (numkem) [reply 2_create_project/scripts/start-stop-status:22]: Thing is, my config is in `/etc/nomad/server.hcl`

Not sure I'd want to ship the config with the package. I know it's not exactly great either way... What about changing the script to check either in the shipped directory or in `/etc/nomad` whichever is there.

The goal for that package was to ship the binary so DSM would play ball with it, the config was done afterwards.

**Pull request closed without merging**